src/diffusers/models/autoencoder_tiny.py

```diff
@@ -137,6 +137,20 @@ def __init__(
         self.latent_shift = latent_shift
         self.scaling_factor = scaling_factor
 
+        self.use_slicing = False
+        self.use_tiling = False
+
+        # only relevant if vae tiling is enabled
+        self.spatial_scale_factor = 2 ** out_channels
+        self.tile_overlap_factor = 0.125
+        # self.tile_sample_min_size = (
+        #     self.config.sample_size[0]
+        #     if isinstance(self.config.sample_size, (list, tuple))
+        #     else self.config.sample_size
+        # )
```

Comment:
I guess this can go out?

Reply:
Oh, I forgot to delete this when copying from vae.

It seems that sd's and sdxl's original vae have different `sample_size` (512 and 1024). So the `tile_size` should match the `sample_size`.

But since taesd's repo doesn't have `sample_size` in config, this doesn't work.